source/dom.js

```diff
@@ -1,4 +1,10 @@
-const div = document.createElement('div');
+import { isBrowserEnv } from './utils';
+
+const fakeWindow = {};
+
+export const global = isBrowserEnv ? window : fakeWindow;
+
+const div = isBrowserEnv ? document.createElement('div') : {};
```

Comment:
I mean we can take away this code into `createNode` method which is not called on server (or at least should not to) so we don't need the condition at all